src/modules/staking.rs

```diff
@@ -147,6 +146,20 @@ fn store_subgraph_deployment_id(events: Events, s: StoreSetString) {
         );
     }
 }
+
+#[substreams::handlers::store]
+fn store_subgraph_deployment_ipfs_hash(events: Events, s: StoreSetIfNotExistsString) {
+    let allocation_created_events = events.allocation_created_events.unwrap();
+
+    for allocation_created in allocation_created_events.allocation_created_events {
+        s.set_if_not_exists(
+            allocation_created.ordinal,
+            &Hex(&allocation_created.subgraph_deployment_id).to_string(),
+            &utils::generate_ipfs_hash(Hex(&allocation_created.subgraph_deployment_id).to_string()),
```

Comment:
Small comment here, in the subgraph we have to do the following to get the correct result:
`deployment.ipfsHash = Bytes.fromHexString(prefix.concat(subgraphID.slice(2))).toBase58()`
Note that the subgraph ID has the first 2 characters removed, that is because in the subgraph, the ID is a hex string, and those hex strings always start with `0x`, so we have to get rid of those for the IPFS hash calculation to be correct.

I'm assuming that in substreams that doesn't happen, and thus why we don't need to do that, but just wanting to confirm that

Reply:
Yeah your assumption is correct, we don't have `0x` in the substreams, I also checked if the hashes turn out correct and they did